docs/Examples2/Image.example.purs

```diff
@@ -7,26 +7,34 @@ import Lumi.Components (($$$))
 import Lumi.Components.Example (example)
 import Lumi.Components.Spacing (Space(..), vspace)
 import Lumi.Components.Text (h4_)
+import Lumi.Components2.Image (ObjectFit(..))
 import Lumi.Components2.Image as Image
 import React.Basic.Classic (JSX)
 
 docs :: JSX
 docs =
-  let flexo = "https://s3.amazonaws.com/lumi-blog/avatars/_x600/flexo.jpg"
+  let flexo = "https://i.picsum.photos/id/985/1000/300.jpg?hmac=t_lmj43iuzwGqZaRMY1ee9udE_pCzfYLgCD49YrCPjw"
+        -- "https://s3.amazonaws.com/lumi-blog/avatars/_x600/flexo.jpg"
   in Array.intercalate (vspace S16)
     [ h4_ "Image default (will respect image's original aspect ratio & dimensions)"
     , example
         $ Image.image
         $$$ flexo
-    , h4_ "Image + resize { width: 120px, height: 40px }, the image will fill the height and width of its parent (object-fit: cover), maintaining its aspect ratio but cropping the image"
+    , h4_ "Image + resize { width: 40px, height: 120px }, the image will fill the height and width of its parent (object-fit: cover), maintaining its aspect ratio but cropping the image"
     , example
         $ Image.image
-        $ Image.resize { width: 120, height: 40 }
+        $ Image.resize { width: 40, height: 120 }
         $$$ flexo
-    , h4_ "Thumbnail default (will always have a square aspect ratio)"
+    , h4_ "Thumbnail default (will always have a square aspect ratio); and defaults to object-fit: contain"
     , example
         $ Image.thumbnail
         $$$ flexo
+    , h4_ "Thumbnail default (will always have a square aspect ratio); but can be override with object-fit: cover"
+    , example
+        $ Image.thumbnail
+        $ _ { content = flexo
+            , objectFit = Cover
```

Comment:
This seems to be the same as just writing
```purescript
Image.thumbnail
$ S.style_ (S.css { objectFit: S.str "cover" })
$$$ flexo
```
Maybe we should use that instead of having two ways of doing the same thing?

If we want to make it easier to do this, maybe we could just provide two style modifiers `Image.cover` and `Image.contain`?

Reply:
This speaks to the principle of preferring the convention of composability over configuration.

Reply:
This won't actually work

```
Image.thumbnail
$ S.style_ (S.css { objectFit: S.str "cover" })
$$$ flexo
```

Because it applies `object-fit: cover` to the Image `E.element R.div'` container rather than the `E.element R.img'` see line 132 in Image.purs re `css: defaultImageStyle theme <> props.css theme`. I was originally going for this approach in FJ (so we wouldn't have to touch lumi-components but realized that would not work).

Alternatively, this approach will work if we move `props.css` to `E.element R.div'` but I need to test this to make sure it works, I _think_ this might be fine tho...

Reply:
I like the use of style modifiers for cover vs. contain, just so we can make it easier to consumer.

Reply:
Ohh I see. Yeah, that might cause unexpected issues.

Reply:
Yeah just straight up moving `props.css` to the `R.img'` causes issues, specifically we lose control over setting the image container's dimensions which is particularly necessary if the image fails to load (shows the placeholder content).

By creating an `imgStyle` prop that can set the `R.img'` elements styles discretely we can accomplish controlling the img element's styles. Also included some style modifiers for thumbnails to make this easier to consume.

Please let me know what you think of this approach, or if there might be an alternative way that's even better than this. Thanks!